src/loaders/compute_tools/tool_common.py

```diff
@@ -584,6 +595,46 @@ def find_gtdbtk_summary_files(output_dir: Path):
     return summary_files
 
 
+def get_filtered_or_failed_genome_ids(output_dir: Path):
```

Comment:
This looks specific to gtdbtk, and so should live in the gtdb specific code

Reply:
Moved to gtdb_tk.py file

Reply:
:+1: